### PR TITLE
CSharp_oop_4 completed

### PR DESCRIPTION
**Pull request by NadzeyaKiz**
The home task CSharp_oop_4 is completed and ready for review.

### CODE REVIEW

File: Code/CSharpOOP4/Piramid.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+        /*if we wont to hardcode the name of the "Piramid" class (so the user will not able to choose unvalid
+         *name) we may write the code in the following form:
+         *public Piramid(double baseArea, double height) : base("Piramid")
+         */
+        public Piramid(string name, double baseArea, double height) : base(name)
+        {
+            _baseArea = baseArea;
+            _height = height;
+        }
```
Comment (MariaEroshevich) [inline Code/CSharpOOP4/Piramid.cs:22]: add space

File: Code/CSharpOOP4/Shape.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CSharpOOP4
+{
+    abstract class Shape
+    {
+        private string _name;
```
Comment (MariaEroshevich) [inline Code/CSharpOOP4/Shape.cs:11]: add space between field and property.

**Review by MariaEroshevich — APPROVED** [review]

File: Code/CSharpOOP4/Shape.cs (hunk 2 above)
Comment (NadzeyaKiz) [reply Code/CSharpOOP4/Shape.cs:11]: done

File: Code/CSharpOOP4/Piramid.cs (hunk 1 above)
Comment (NadzeyaKiz) [reply Code/CSharpOOP4/Piramid.cs:22]: done

**Pull request merged**